packages/jest-runner/src/runTest.ts

```diff
@@ -150,16 +158,17 @@ async function runTestInternal(
 
   const start = Date.now();
 
-  const sourcemapOptions = {
+  const sourcemapOptions: sourcemapSupport.Options = {
     environment: 'node',
     handleUncaughtExceptions: false,
-    retrieveSourceMap: source => {
+    retrieveSourceMap: (source: string) => {
       const sourceMaps = runtime && runtime.getSourceMaps();
       const sourceMapSource = sourceMaps && sourceMaps[source];
 
       if (sourceMapSource) {
         try {
           return {
+            // @ts-ignore TODO: clarify in PR
```

Comment:
Not sure how to resolve this. Some help would be appreciated

Reply:
resolve what? What's the error?

Reply:
`JSON.parse` expects a string as an argument but `fs.readFileSync` returns a buffer or a string. Casting the return type of the readfile call returns this error `Conversion of type 'Buffer' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.ts(2352)` I guess just `ts-ignore this for now?`

Reply:
Fixed it by adding encoding to `readFile` so it returns a string. I didn't know `JSON.parse` could take a buffer (it obviously worked previously), but we should use a string here